blocks/library/quote/index.js

```diff
@@ -243,9 +245,40 @@ registerBlockType( 'core/quote', {
 					<p key={ i }>{ paragraph.children && paragraph.children.props.children }</p>
 				) ) }
 				{ citation && citation.length > 0 && (
-					<footer>{ citation }</footer>
+					<cite>{ citation }</cite>
 				) }
 			</blockquote>
 		);
 	},
+
+	deprecatedVersions: [
+		{
+			attributes: {
+				...blockAttributes,
+				citation: {
+					type: 'array',
+					source: 'children',
+					selector: 'footer',
+				},
+			},
+
+			save( { attributes } ) {
```

Comment:
I have a slight concern with accumulating `save` mechanisms just for validation, but I don't see an alternative. Might be worth moving all the deprecated definitions into a `deprecated.js` file to not burden the main js with additional save definitions.

Reply:
What about the "attributes"? do you think we should share common attributes or maybe fine do duplicate?